pkg/authorization/registry/resourceaccessreview/rest.go

```diff
@@ -63,8 +65,19 @@ func (r *REST) Create(ctx apirequest.Context, obj runtime.Object, _ bool) (runti
 	}
 
 	attributes := util.ToDefaultAuthorizationAttributes(nil, resourceAccessReview.Action.Namespace, resourceAccessReview.Action)
-	users, groups, err := r.subjectLocator.GetAllowedSubjects(attributes)
-
+	subjects, err := r.subjectLocator.AllowedSubjects(attributes)
+
+	//TODO I think that somehow should reuse the code from authorizer#GetAllowedSubjects
+	users := sets.String{}
+	groups := sets.String{}
+	for _, subject := range subjects {
+		switch subject.Kind {
+		case rbac.UserKind:
```

Comment:
need to handle ServiceAccount and map to a username

Reply:
@liggitt I was wondering if we should take the code from here https://github.com/openshift/origin/pull/16034/files#diff-5654e020d459e05c8fcca57d5675ece4R59 and move to an utility function. In this way would be possible to reuse here and also at `reviewer.go` 
Does it make sense? If yes, which place would be the best for it?

Reply:
sure, a utility function to go from subjects to users/groups is fine